app/helpers/application_helper.rb

```diff
@@ -119,7 +119,7 @@ def sanitized_sidebar(text)
                                             tags: %w[p b i em strike strong u br]
   end
 
-  def follow_button(followable, style = "full", classes = "")
+  def follow_button(followable, style = "full", classes = "", aria_label: "")
```

Comment:
refactor (blocking) - I think it would be even better if we didn't have to worry about passing in the aria-label, given it is always going to take the format of `"Follow #{type_of_the_followable}: #{followable_name}"`

Can we update this helper function to dynamically create the correct aria-label based on the type and name of the followable?

Reply:
Why didn't I think of it 🤦🏻